app/views/ReactionsView/index.js

```diff
@@ -0,0 +1,111 @@
+import React, { useEffect } from 'react';
+import {
+	View, Text, TouchableOpacity
+} from 'react-native';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { withSafeAreaInsets } from 'react-native-safe-area-context';
+import ScrollableTabView, { ScrollableTabBar } from 'react-native-scrollable-tab-view';
+
+import I18n from '../../i18n';
+import { withTheme } from '../../theme';
+import { themes } from '../../constants/colors';
+import { withDimensions } from '../../dimensions';
+import sharedStyles from './styles';
+import SafeAreaView from '../../containers/SafeAreaView';
+import Emoji from '../../containers/message/Emoji';
+
+import TabPage from './renderPage';
+
+const scrollProps = {
+	keyboardShouldPersistTaps: 'always',
+	keyboardDismissMode: 'none'
+};
+
+const RenderLabel = React.memo(({
+	name, baseUrl, getCustomEmoji, reactions, page, theme
+}) => (
+	<View style={sharedStyles.tabView}>
+		<Emoji
+			content={name}
+			standardEmojiStyle={sharedStyles.reactionEmoji}
+			customEmojiStyle={sharedStyles.reactionCustomEmoji}
+			baseUrl={baseUrl}
+			getCustomEmoji={getCustomEmoji}
+		/>
+		<Text style={[sharedStyles.textBold, { color: themes[theme].bodyText }]}>{reactions[page].usernames.length}</Text>
+	</View>
+));
+
+const renderTab = (getCustomEmoji, baseUrl, reactions, theme) => (
+	name, page, isTabActive, onPressHandler, onLayoutHandler
+) => (
+	<TouchableOpacity
+		activeOpacity={0.7}
+		testID={isTabActive ? `reaction-tab-${ name }-active` : `reaction-tab-${ name }`}
+		key={`${ name }_${ page }`}
+		onPress={() => onPressHandler(page)}
+		onLayout={onLayoutHandler}
+		underlayColor={themes[theme].tintActive}
+	>
+		<RenderLabel getCustomEmoji={getCustomEmoji} reactions={reactions} baseUrl={baseUrl} name={name} page={page} theme={theme} />
+	</TouchableOpacity>
+);
+
+const ReactionsView = React.memo(({
+	navigation, theme, route, baseUrl, customEmojis
+}) => {
+	const reactions = route.params?.reactions || [];
+
+	const getCustomEmoji = (name) => {
+		const emoji = customEmojis[name];
+		if (emoji) {
+			return emoji;
+		}
+		return null;
+	};
+
+	useEffect(() => {
+		navigation.setOptions({ title: I18n.t('Reactions') });
+	}, []);
+
+	return (
+		<View style={[sharedStyles.container, { backgroundColor: themes[theme].backgroundColor }]}>
```

Comment:
Do we need this?

Reply:
Mostly for sanity and inheriting any container changes later